pkg/cli/alpha/config-gen/testdata/enableprometheus/expected.yaml

```diff
@@ -9,7 +9,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: (unknown)
+    controller-gen.kubebuilder.io/version: ""
```

Comment:
We should dig into this a little bit. IIRC there's some code in controller-gen that knows how to auto-detect the version of the binary that's running. This may have changed in 1.18 with the changes that came in with auto-embedding version information as part of `go build`.

Reply:
Hi @joelanford 
I'm not sure I'm following here - how's that related to this "(unknown)," and why is it important to understand how it gets serialized?

Reply:
+1 for @joelanford comments. 
It seems valid we deeger a little to understand it to be sure that it is right and we are not breaking anything.

Reply:
see https://github.com/kubernetes-sigs/controller-tools/pull/662